app/Website.php

```diff
@@ -201,8 +201,6 @@ public function getShowLinkAttribute()
      */
     public function setUrlAttribute($value)
     {
-        $parts = parse_url($value);
-
-        $this->attributes['url'] = sprintf('%s://%s', $parts['scheme'], $parts['host']);
```

Comment:
Hi, I'm not sure about this one - The rest of the application "relies" on the data being normalised before it gets to the database.

Removing this normalisation means that other areas would receive unexpected input e.g.

If the user enters `https://google.com/` then certain areas might generate inaccurate urls e.g. `https://google.com//favicon.ico`

Reply:
I see your point.
Would you prefer to add a normalized_hostname property (just like the dns_hostname) that can be used in cases where a normalized hostname is required, and using the original user-input for open-graph/uptime/crawler  checks?


But when testing i didn´t encounter any issues with wrongly generated urls. Only the robots check had issues, but that is taken care of in this change.